Server/JokesOnYou.Web.Api/Services/JokesService.cs

```diff
@@ -47,7 +46,7 @@
             var saved = await _unitOfWork.SaveAsync();
             if (!saved)
             {
-                if(await _jokesRepo.GetJokeDtoAsync(joke.Id) == null)
```

Comment:
You can remove this since: If the save failed, theres no reason to try and get the item from the db thats just another request made to the db

Reply:
This wasn't my code. My OCD just had to add that space after the if statement. But following the boy-scout-rule I will fix it anyhow :)

Reply:
I thought we agreed that its possible some other method saved this change?
then this would fail and require a second check?